landing_page/spec/factories/microposts.rb

```diff
@@ -0,0 +1,10 @@
+FactoryBot.define do
+  factory :micropost do
+    title { 'Well hello there!' }
+    content do
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+                 Proin maximus pellentesque lacus, nec aliquam mi auctor id.'
+    end
+    id { '12' }
```

Comment:
Can you think of a situation where having the id set to a constant might not be great?

Reply:
Would it be something like if a person modified the function to return a different id, it could fail?

Reply:
I was more thinking along the lines of what happens if you try to create two microposts via the factory in a single test.